CHANGELOG.md

```diff
@@ -3,6 +3,37 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.19.6
+
+_Feb 29, 2024_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+- 🌍 Improve Korean (ko-KR) locale on the Data Grid (#11972) @Luzi
+- 🌍 Improve Chinese (zh-CN) locale on the Date Pickers (#12250) @headironc
+
+### Data Grid
+
+#### `@mui/x-data-grid@6.19.6`
+
+- [l10n] Improve Korean (ko-KR) locale (#12186) @Luzi
```

Comment:
```suggestion
Nothing this time.
```
This is an item that belongs under the `@mui/x-date-pickers` section that hasn't even been constructed due to the limitations of the script. 🙈